.github/ISSUE_TEMPLATE/github_repo_request.md

```diff
@@ -57,4 +57,8 @@ Fill out the following fields. For security purpose, please provide the followin
 * GitHub Team to Join (if exists): 
 
 
+## Step 4
+Once you have a private repo and could like to invite more team members to work on it, please create a `Request for BCGOV-C Private GitHub Organization Membership`!
```

Comment:
typo in the above instruction. I think it should be "and would like" vs "and could like"